src/datepicker/datepicker-month-view.scss

```diff
@@ -20,6 +20,7 @@ ngb-datepicker-month-view {
   &-weekdays {
     border-bottom: 1px solid rgba(0, 0, 0, 0.125);
     border-radius: 0;
+    background-color: var(--light);
```

Comment:
Using 'var' was a good idea, but unfortunately we can't use this functionality as ng-bootstrap supports IE >= 10

Reply:
var is already in use:

  &-weekday {
    color: #5bc0de;
    color: var(--info);
  }

Do it like this maybe better?